tao_compiler/mlir/xla/ral/BUILD

```diff
@@ -161,15 +162,31 @@ cc_library(
     deps = if_platform_alibaba([":real_disc_patine_client_deps"])
 )
 
+cc_library(
+    name = "blade_gemm_deps",
+    deps = if_torch_disc(
```

Comment:
What's the difference between `@blade_gemm` and `//tao/blade_gemm`, I am a little bit confused...
And why use `@blade_gemm` when torch_disc is `ON` ?

Reply:
`//tao/blade_gemm` is from a `BUILD` files write for cmake-prebuild blade gemm lib and headers build by python scripts, `@blade_gemm` is pure bazel target which doesn't depend on any pre-bazel cmake actions.

Reply:
`//tao/blade_gemm` is from a `BUILD` files write for cmake-prebuild blade gemm lib and headers build by python scripts, `@blade_gemm` is pure bazel target which doesn't depend on any pre-bazel cmake actions.

Reply:
`if_torch_disc` is better to be eliminated, it's better if there's nothing special for any of torch or tf.